applications/jonny5/src/j5_channels.erl

```diff
@@ -328,6 +328,27 @@ per_minute_cost(AccountId) ->
                         call_cost(Channel) + Cost
                 end, 0, ets:select(?TAB, MatchSpec)).
 
+-spec real_per_minute_cost(ne_binary()) -> non_neg_integer().
+real_per_minute_cost(AccountId) ->
+    MatchSpec = [{#channel{account_id = AccountId
+                           ,account_billing = <<"per_minute">>
+                           ,_='_'
+                          }
+                  ,[]
+                  ,['$_']
+                 }
+                 ,{#channel{reseller_id = AccountId
+                            ,reseller_billing = <<"per_minute">>
+                            ,_='_'
+                           }
+                   ,[]
+                   ,['$_']
+                  }
+                ],
+    lists:foldl(fun(Channel, Cost) ->
```

Comment:
I'm getting back to this, apologies for the delay. I'm going to write some test code to compare the two and will let you know!